config/jobs/cert-manager/cert-manager-presubmits.yaml

```diff
@@ -450,6 +450,73 @@ presubmits:
         options:
         - name: ndots
           value: "1"
+
+  # An E2E test job to allow us to manually trigger the Venafi TPP E2E tests
+  # with the following GitHub comment:
+  #
+  #  /test pull-cert-manager-e2e-v1-20-venafi-issuer-tpp
+  #
+  # See https://github.com/jetstack/cert-manager/issues/3555
+  #
+  - name: pull-cert-manager-e2e-v1-20-venafi-issuer-tpp
+    cluster: gke
+    context: pull-cert-manager-e2e-v1-20-venafi-issuer-tpp
+    always_run: false
+    optional: true
+    max_concurrency: 4
+    agent: kubernetes
+    decorate: true
+    branches:
+    - master
+    - release-1.3
+    annotations:
+      description: Runs the E2E tests labelled [Feature:VenafiIssuer:TPP] against a Kubernetes v1.20 cluster
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-bazel-remote-cache-enabled: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-venafi-tpp-credentials: "true"
+      preset-retry-flakey-tests: "true"
+    spec:
+      containers:
+      - image: eu.gcr.io/jetstack-build-infra-images/bazelbuild:20200428-6b034c1-2.2.0
+        args:
+        - runner
+        - devel/ci-run-e2e.sh
+        - -ginkgo.focus
+        - '\[Feature:VenafiIssuer:TPP\]'
```

Comment:
We should consider the format we want to use for Features here in our test names (e.g. `Feature:Issuers:VenafiTPP` to allow matching `Feature:Issuers:*` or something)